PyPrograms/areas/areas.py

```diff
@@ -0,0 +1,25 @@
+
+
+# it uses GPL V3 LICENSE
+# what we are going to do
+print("We can find areas of different shapes from below")
+
+# shapes we have
+print("1. Triangle")
+print("2. Square")
+
+# take input
+sipt = input("For what you would like to find area?")
+
+# if 1 is sel
+if sipt == "1" :
+    print("coming soon")
```

Comment:
tell user that this is WIP from the start and in the readme

Reply:
WIP ??

Reply:
Ok WORK IN PROGRESS